packages/release/proxy-env

```diff
@@ -0,0 +1,10 @@
+{{#if settings.network.http-proxy}}
+HTTP_PROXY={{settings.network.http-proxy}}
+http_proxy={{settings.network.http-proxy}}
+{{/if}}
+{{#if settings.network.https-proxy}}
+HTTPS_PROXY={{settings.network.https-proxy}}
+https_proxy={{settings.network.https-proxy}}
```

Comment:
If we're targeting Go programs, either the capitalized or lower-case environment variable will work.  It should be fine to just use `HTTP_PROXY`/`HTTPS_PROXY` without duplication.

Reply:
I wish there is some universal standard for how these should be named and specified. I think it would be best to specify both just in case if we ever have another non-Go based service use these environment variables.